spec/cart_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'cart'
+
+describe Cart do
+
+end
```

Comment:
Of course, another argument is: you don't need a `Cart` class because by now it's basically an `Array` instance.